x-pack/test/performance/config.playwright.ts

```diff
@@ -31,21 +33,15 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
       env: {
-        ELASTIC_APM_ACTIVE: 'true',
+        ELASTIC_APM_ACTIVE: process.env.DISABLE_APM === 'true' ? 'false' : 'true',
```

Comment:
Can we still use `ELASTIC_APM_ACTIVE` and grab it from process.env here?

Reply:
Yeap definitely 👍🏼